src/client/src/ui/common/popout/reactPopoutService.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { PopoutOptions } from './';
+import './popoutRegion.scss';
+
+let POPOUT_CONTAINER_ID = 'container_id';
+
+export default class ReactPopoutService {
+  openPopout(options:PopoutOptions, view:React.Component) {
+    let popoutContainer;
+    let windowOptionsString = this._getWindowOptionsString(options.windowOptions);
+    let childWindow = window.open(options.url, options.title, windowOptionsString);
+    let onloadHandler = () => {
+      childWindow.document.title = options.title;
+      popoutContainer = childWindow.document.createElement('div');
+      popoutContainer.id = POPOUT_CONTAINER_ID;
+      childWindow.document.body.appendChild(popoutContainer);
```

Comment:
Ok good Idea, lets chat in person to to sort it. Might be quicker to code it up together